### ISSUE TITLE
Shrinkwrap in release

### ISSUE THREAD

**Issue by Enteee** [balderdashy/sails-postgresql#269]
Overwriting dependencies of this adapter using npm-shrinkwrap seems to be impossible since you added a complete shrinkwrap file to the 0.12.x - release. Is there a reason for the shrinkwrap file beeing there? If not, i'd suggest removing it again.

**Comment by sailsbot** [comment]
@Enteee Thanks for posting, we'll take a look as soon as possible.  In the meantime, if you haven’t already, please carefully read the [issue contribution guidelines](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md#ii-issue-contributions) and double-check for any missing information above.  In particular, please ensure that this issue is about a stability or performance bug with a  documented feature; and make sure you’ve included detailed instructions on how to reproduce the bug from a clean install.   Finally, don’t forget to include the version of Node.js you tested with, as well as your version of Sails or Waterline, and of any relevant standalone adapters/generators/hooks.

Thank you!

**Comment by sgress454** [comment]
@Enteee can you clarify why it is you're trying to override the dependencies?  Is there an important patch to one of the dependencies of `sails-postgresql`?

**Comment by Enteee** [comment]
@sgress454 As packing a shrinkwrap is afaik bad practice i do think that the why should not be of importance. But i'd still like to give you some insights. It's mostly because of:
* https://github.com/balderdashy/waterline-sequel/pull/100
* https://github.com/balderdashy/waterline-sequel/issues/101

But maybe this issue resolves itself if you have some profound resons for releasing versions with a shrinkwrap.

**Comment by particlebanana** [comment]
Hey @Enteee we included the shrinkwrap file because we were having issues with dependencies of modules even when we had our own dependencies pinned. Using the shrinkwrap allowed us to guarantee that the module worked the way we expected it to. I'm not sure if thats bad practice or not but it was helpful for us.

If there are no longer issues and the test continue to pass, I can remove the `shrinkwrap.json` file if needed from the stable branch which is currently the `0.11` branch.

As part of the upgrade to Sails 1.0 and Waterline 0.13 we have abandoned the use of rolling our own sql generator in favor of using [Knex](http://knexjs.org/) which is much more widely supported and frees us from having to handle that part of the query process. We also have a new lower level query syntax called statements that support the kind of `groupBy` clauses you are attempting to perform as seen [here](https://github.com/treelinehq/waterline-sql-builder/blob/e396956ad286cc28a5aca586df727c2140dcc372/test/unit/generate/groupBy.test.js#L7-L11).

By using that syntax and Knex to build parameterized statements they should be safe from injections.
